kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -23,16 +23,24 @@ The script assumes that either the image, `container-registry.oracle.com/middlew
 
 ```
 $ ./create-rcu-schema.sh -h
-usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <rcuType> -d <dburl> -i <image> -s <dockerstore> [-h]
-  -s RCU Schema Prefix (required)
+usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <schemaType> -d <dburl> -i <image> -p <docker-store> -n <nameSpace> -q <sysPassword> -r <schemaPassword>  -o <rcuOutputDir>  [-h]
+  -s RCU Schema Prefix (needed)
```

Comment:
we use "required" not "needed"

Reply:
create-rcu-schema.sh was using this. Updated create-rcu-schema.sh and README.md with the required changes.